src/homework/homework02/Employee.java

```diff
@@ -3,23 +3,32 @@
 import homework.homework02.Person;
 
 public class Employee extends Person {
```

Comment:
Relacja Employee vs Person 👎
Opiszę przy relacji Student vs Person, bo tam jest to samo